packages/client/components/NewMeetingTeamPickerAvatars.tsx

```diff
@@ -56,12 +53,8 @@ const NewMeetingTeamPickerAvatars = (props: Props) => {
 
   const {teamMembers} = team
 
-  if (!teamMembers) {
-    return null
-  }
-
   const randomAvatars = useMemo(() => {
-    const randomTeamMembers = getShuffledArr(teamMembers)
+    const randomTeamMembers = getShuffledArr(teamMembers as TeamMember[])
```

Comment:
-1 teamMembers is not resolved as nullable for me? If it is for you, please change to `teamMembers ?? []`

Reply:
@Dschoordsch it is not nullable, but without specifying that type it complains about 
```
  The type 'readonly { readonly id: string; readonly picture: string; }[]' is 'readonly' and cannot be assigned to the mutable type '{ readonly id: string; readonly picture: string; }[]'.
  ```

Reply:
Then let's fix the signature of `getShuffledArr` as it does a `slice` at the beginning anyways.

Reply:
Fixed